arbor/iexpr.cpp

```diff
@@ -281,6 +281,23 @@ struct exp: public iexpr_interface {
     iexpr_ptr value;
 };
 
+struct step: public iexpr_interface {
+    step(iexpr_ptr v): value(std::move(v)) {}
+
+    double eval(const mprovider& p, const mcable& c) const override {
+        double x = value->eval(p, c);
+        if (x > 0) {
+            return 1.0;
+        } else if (x < 0) {
+            return 0.0;
+        } else {
+            return 0.5;
+        }
```

Comment:
replacing branches with operations, very obscure
```suggestion
        // x <  0:  0
        // x == 0:  0.5
        // x >  0:  1
        return 0.5*((0. < x) - (x < 0.) + 1);
```

Reply:
This is also more in line with the 'official' NeuroML java (sign) implementation as opposed to the Python (if statements)